src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -419,6 +419,14 @@ public void MetricExtractorAfterSamplingVerbose(string appDomainName = "Incorrec
             this.WriteEvent(34, this.nameProvider.Name);
         }
 
+        [Event(35,
+            Message = "At least one telemetry channel cannot consume incoming telemetry fast enough. Some telemetry was dropped to prevent out of memory condition.",
```

Comment:
Good suggestions, but... this event became unnecessary after the last round of changes. I just missed it when doing the refactoring.

I have pushed a commit that removes the new event entirely.